src/packages/@ncigdc/modern_components/GenesTable/GenesTable.tsx

```diff
@@ -13,12 +12,79 @@ import { CreateExploreGeneSetButton } from '@ncigdc/modern_components/withSetAct
 import { AppendExploreGeneSetButton } from '@ncigdc/modern_components/withSetAction';
 import { RemoveFromExploreGeneSetButton } from '@ncigdc/modern_components/withSetAction';
 import timestamp from '@ncigdc/utils/timestamp';
+import { IColumnProps } from '@ncigdc/tableModels/utils';
+import { IGroupFilter } from '@ncigdc/utils/filters/types';
 
-import tableModel from './GenesTable.model';
 import { theme } from '@ncigdc/theme';
 import withSelectIds from '@ncigdc/utils/withSelectIds';
+export interface ITotalNumber {
+  hits: {
+    total: number;
+  };
+}
 
-export default compose(
+export interface INodeProps {
+  node: {
+    gene_id: string;
+    id: string;
+    symbol: string;
+    name: string;
+    cytoband: string[];
+    biotype: string;
+    numCases: number;
+    is_cancer_gene_census: boolean;
+    ssm_case: ITotalNumber;
+    cnv_case: ITotalNumber;
+    case_cnv_gain: ITotalNumber;
+    case_cnv_loss: ITotalNumber;
+  };
+}
+interface IGenesProps {
+  hits: {
+    total: number;
+    edges: INodeProps[];
+  };
+}
+interface IGenesTableProps {
+  genesTableViewer: {
+    explore: {
+      genes: IGenesProps | undefined;
+      filteredCases: ITotalNumber | undefined;
+      cases: ITotalNumber | undefined;
+      cnvCases: ITotalNumber | undefined;
+    };
+  };
+  filters: IGroupFilter;
+  relay: { route: { params: { [x: string]: any } } };
+  setSurvivalLoadingId: string;
+  survivalLoadingId: string;
+  setSelectedSurvivalData: ({}) => void;
+  selectedSurvivalData: any;
+  hasEnoughSurvivalDataOnPrimaryCurve: boolean;
+  context: string;
+  query: { [x: string]: any };
+  ssmCounts: { [x: string]: number };
+  ssmCountsLoading: boolean;
+  parentVariables: {
+    genesTable_offset: number;
+    genesTable_size: number;
+    [x: string]: any;
+  };
+  tableColumns: Array<IColumnProps<boolean>>;
+  selectedIds: string[];
+  setSelectedIds: (props: string[]) => void;
+  sort: any;
```

Comment:
If this is using the same sort as the rest of the app there are sort types available